mmv1/products/gkebackup/BackupPlan.yaml

```diff
@@ -441,6 +458,13 @@ properties:
                   required: true
                   description: |
                     The name of a Kubernetes Resource.
+      - !ruby/object:Api::Type::Boolean
+        name: permissiveMode
+        description: |
+          This flag specifies whether Backups will not fail when
+          Backup for GKE detects Kubernetes configuration that is
+          non-standard or requires additional setup to restore.
+        default_from_api: false
```

Comment:
`default_from_api: false` is not needed here, as it the default value is false.

Reply:
Done